### PR TITLE
GaxStation, a new lowpop map that plays into being lowpop

### PR DESCRIPTION
**Pull request by ToasterBiome**
# Document the changes in your pull request

Basically: I love KiloStation maintenance. I hate everything else about the map. It's so visually cluttered, things are cramped, maintenance is like half the station so if you die you're thrown in there and forgotten for the rest of the round. I also LOVE BoxStation. I love how consistent it is. I love the design style of new Medbay. I love how every department gets their own nook. Which leads to:

GaxStation is a lowpop (0-30) station modelled after a ship (with a follower AI ship) the likes of Atlas from other servers. Departments are clear and compact, while leaving open space for busy areas (Medbay and Security).

Medbay is centered prominently in the middle, with entrances on three sides (and maintenance on another) letting every slain body found be easily returned. Civilian areas such as Tool Storage, Dorms, and Fitness Room are peppered around the map to encourage moving around the station. Many departments have a foyer of their own to interact with the people inside, notably Science. Service is condensed into one part of the map, with a nearly shared maintenance like box or meta. Command is at the front of the ship, while Engineering is at the back, sharing it's vertical space with Security and Cargo.

Maintenance is _much, much_ smaller, accounting for a tiny portion of this map. They're more akin to the cramped maintenance tunnels that would be on a ship. This helps force things onto the main areas of the station, where you have to do more risky plays on lowpop since the ship is EMPTY. Unfortunately, this small and irregular size means that I do not have any space for random maintenance. I love random maintenance, and I think they add a lot to the character, but maintenance is definitely taking a backseat so departments/hallways can shine.

You get to the AI Satellite either through space or a teleporter that shares space with the vault. Another teleporter may be added but I do not want to include transit tubes between two "ships".

Removed areas:

I thought long and hard about what can stay and what can go, and some things just take up way too much space to be worthwhile to include on a lowpop station. Other than the holodeck being used _one_ time in me playing on KiloStation, the others are completely unused.

- Holodeck
- Aux Base Construction
- Experimentor
- Testing Lab
- Construction Area
- Courtroomᵃ
- Clerk's Officeᵃ
- Psych's Officeᵃ

ᵃ if enough people want these, I can try to fit these areas in, I have some ideas but not many.

Onto some pictures, if you'd like to request some zoomed in pictures, just tell me the parts you want until I find a better way to show it.

Main Station:
image

Labelling: 

image

AI/Telecommunications Satellite (The Whale)
image

Size comparison related to other maps:

Gax:
image

Box:
image

Kilo:
image

Meta:
image

The entire map is ready to be play-tested as is, but there are many more things that have to be added or changed, I simply wanted this PR to be made so that I can gauge player reaction/see what has to be emergency changed before a test merge.

- [ ] Fix bug where cargo shuttle wants to default to box's (currently works, just  destroys some stuff around it whoops)
- [x] AI sat needs more power generation
- [x] Lots of signage pointing to different departments
- [x] make sure there are light switches everywhere
- [x] make sure there are fire alarms everywhere
- [x] add more fluff to maints
- [x] add more fluff/random shit to each department
- [x] Random event/Xeno spawns
- [x] Carp spawns
- [x] Decide if clerk/psychiatrist can be fit into the map

# Wiki Documentation

A whole entire map!

# Changelog

:cl:  
rscadd: Adds GaxStation
/:cl:

### CODE REVIEW

**Comment by Mqiib** [comment]
Cool as heck dude

**Comment by nmajask** [comment]
Needs more telescreens

**Comment by cowbot92** [comment]
It takes a lot of work to build a map, glad to see the work you've put in.

**Comment by patpol4** [comment]
cool name

File: _maps/gaxstation.dm [hunk 1]
```diff
@@ -0,0 +1 @@
+#define FORCE_MAP "_maps/gaxstation.json"
```
Comment (MenacingManatee) [inline _maps/gaxstation.dm:1]: Don't forget to remove this :p

**Review by MenacingManatee — COMMENTED** [review]
Jamie is probably gonna want you to remove those horror file changes too

**Comment by ToasterBiome** [comment]
> Jamie is probably gonna want you to remove those horror file changes too

Yes the line endings make these difficult, talking to other maintainers to see how i can fix it.

File: _maps/gaxstation.dm (hunk 1 above)
Comment (ToasterBiome) [reply _maps/gaxstation.dm:1]: image

every map has a .dm hook to the json file :)

**Comment by bruhlookatthisdood** [comment]
he turned the ai satellite into an ai ship

**Comment by cuackles** [comment]
Remember to add Gary the snail too :)

**Comment by Marmio64** [comment]
no monastery ship?
edit: also try adding a shuttle that can travel go between places. Pubby had a shuttle that could go from the monastery to dorms on the mainship.

**Comment by Cartlord** [comment]
What would be the gain from adding such a shuttle?

**Comment by Marmio64** [comment]
> What would be the gain from adding such a shuttle?

so its easier to get to the ai ship without having to float into space without guidance. Im sure lots of people will be asking where the ai satelite is and having a shuttle that can just take you there easily if the teleporter is out would be nice. Plus, it's just cool and would play into the theme of having shuttles and spacecraft.

**Comment by bruhlookatthisdood** [comment]
> > What would be the gain from adding such a shuttle?
> 
> so its easier to get to the ai ship without having to float into space without guidance. Im sure lots of people will be asking where the ai satelite is and having a shuttle that can just take you there easily if the teleporter is out would be nice. Plus, it's just cool and would play into the theme of having shuttles and spacecraft.

should be in sec or command.
reminds me of hte old ship that would take you from sec to the perma station.

**Comment by ToasterBiome** [comment]
> > What would be the gain from adding such a shuttle?
> 
> so its easier to get to the ai ship without having to float into space without guidance. Im sure lots of people will be asking where the ai satelite is and having a shuttle that can just take you there easily if the teleporter is out would be nice. Plus, it's just cool and would play into the theme of having shuttles and spacecraft.

this is a good idea and an alternative to having a big tube. The only problem I could see with this is an annoying AI spamming the console.

**Comment by bruhlookatthisdood** [comment]
> > > What would be the gain from adding such a shuttle?
> > 
> > 
> > so its easier to get to the ai ship without having to float into space without guidance. Im sure lots of people will be asking where the ai satelite is and having a shuttle that can just take you there easily if the teleporter is out would be nice. Plus, it's just cool and would play into the theme of having shuttles and spacecraft.
> 
> this is a good idea and an alternative to having a big tube. The only problem I could see with this is an annoying AI spamming the console.

then maybe make a console that ai cant access similar to disabling cyborg access on the ai revamped console or whatever (good luck with that)
AI fuck with their transit tube without deconstructing it, they shouldn't be able to touch the shuttle without deconstructing it too.

**Comment by ToasterBiome** [comment]
> Needs more telescreens

Feel free to add some :)

**Comment by bruhlookatthisdood** [comment]
The AIs solars are set up round start? Sucking the soul out of engiborgs.

**Comment by Moja10** [comment]
I like this a lot, way better than Kilo. Looks actually reasonable. Would like it added. Also HoS Office window is hallway facing instead of space facing MonkaS

**Comment by Moja10** [comment]
In regards to not including Clerk and Psych offices, I would say those are necessary just because they are jobs that people can role during lowpop and quite frankly can't do anything without their office. They can't do much as it is but if someone was a Clerk, they would have to hijack a random office and call it clerk office or some space like dorms, which is less than ideal, and this map already strips the other unused rooms like construction so that's not an option either.

**Comment by ToasterBiome** [comment]
> In regards to not including Clerk and Psych offices, I would say those are necessary just because they are jobs that people can role during lowpop and quite frankly can't do anything without their office. They can't do much as it is but if someone was a Clerk, they would have to hijack a random office and call it clerk office or some space like dorms, which is less than ideal, and this map already strips the other unused rooms like construction so that's not an option either.

If I didn’t include the offices then I would not include the jobs, we have the ability to turn them off per map

**Comment by JamieD1** [comment]
Will review

**Comment by ToasterBiome** [comment]
image

Clerk

**Comment by JamieD1** [comment]
Map reviewed with Author, changes ongoing

**Review by JamieD1 — APPROVED** [review]
Bridge fucking sucks

**Pull request merged**